panic: slices.MaxFunc: empty list on dbc info

I just ran into this and haven't troubleshooted at all:

```console
$ dbc search postgres
• postgresql - An ADBC driver for PostgreSQL developed under the Apache Software Foundation [installed: user=>1.8.0]

~
$ dbc info postgres
Error: driver `postgres` not found in driver registry index
panic: slices.MaxFunc: empty list

goroutine 1 [running]:
slices.MaxFunc[...](...)
	/home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.6.linux-amd64/src/slices/sort.go:111
github.com/columnar-tech/dbc.Driver.MaxVersion({{0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0, ...}, ...})
	/home/runner/work/dbc/dbc/drivers.go:362 +0x124
main.formatDriverInfo({{0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0, ...}, ...})
	/home/runner/work/dbc/dbc/cmd/dbc/info.go:66 +0x88
main.infoModel.FinalOutput(...)
	/home/runner/work/dbc/dbc/cmd/dbc/info.go:95
main.main()
	/home/runner/work/dbc/dbc/cmd/dbc/main.go:209 +0x4b0
$ dbc --version
v0.1.0
```

--- a/cmd/dbc/info.go
+++ b/cmd/dbc/info.go
@@ -63,6 +63,10 @@ func (m infoModel) Init() tea.Cmd {
 }
 
 func formatDriverInfo(drv dbc.Driver) string {
+	if len(drv.PkgInfo) == 0 {
+		return ""
+	}
+
 	info := drv.MaxVersion()
 	var b strings.Builder
 

--- a/cmd/dbc/info_test.go
+++ b/cmd/dbc/info_test.go
@@ -26,3 +26,11 @@ func (suite *SubcommandTestSuite) TestInfo() {
 		"   - linux_amd64\n   - macos_amd64\n"+
 		"   - macos_arm64\n   - windows_amd64\n", out)
 }
+
+func (suite *SubcommandTestSuite) TestInfo_DriverNotFound() {
+	m := InfoCmd{Driver: "non-existent-driver"}.
+		GetModelCustom(baseModel{getDriverList: getTestDriverList, downloadPkg: downloadTestPkg})
+	out := suite.runCmdErr(m)
+
+	suite.validateOutput("Error: driver `non-existent-driver` not found in driver registry index\r\n\r ", "", out)
+}

